evm-contracts/test/v0.6/PrepaidAggregator.test.ts

```diff
@@ -1679,4 +1681,149 @@ describe('PrepaidAggregator', () => {
       matchers.bigNum(originalBalance.add(deposit), newBalance)
     })
   })
+
+  describe('#startNewRound', () => {
+    beforeEach(async () => {
+      await aggregator
+        .connect(personas.Carol)
+        .addOracle(personas.Neil.address, personas.Neil.address, 1, 1, 0)
+
+      await aggregator.connect(personas.Neil).updateAnswer(nextRound, answer)
+      nextRound = nextRound + 1
+
+      await aggregator.setAuthorization(personas.Carol.address, true)
+
+      await aggregator.connect(personas.Carol)
+    })
+
+    it('announces a new round via log event', async () => {
+      const tx = await aggregator.startNewRound()
+      const receipt = await tx.wait()
+      const event = matchers.eventExists(
+        receipt,
+        aggregator.interface.events.NewRound,
+      )
+
+      assert.equal('NewRound(uint256,address,uint256)', event.eventSignature)
```

Comment:
Isn't this captured in the match to `aggregator.interface.events.NewRound`? It seems to be introducing an unnecessary dependency in the test. (If you update the log signature in the contract, you have to change the signature in the test here, too.)